SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+SerenityOS is a hobby operating system developed by volunteers. It is not used anywhere in production (yet!), and so bugs and vulnerabilities can be safely disclosed publicly. If you find an issue, we would prefer if you report it as a [GitHub issue](https://github.com/SerenityOS/serenity/issues/new).
```

Comment:
```suggestion
SerenityOS is a hobby operating system developed by volunteers. It's still in early development so bugs and vulnerabilities can be safely disclosed publicly. If you find an issue, we would prefer if you report it as a [GitHub issue](https://github.com/SerenityOS/serenity/issues/new).
```

I don't think "not used anywhere in production" is factual(depending on the definition) and diminishes the systems great achievements so why not just remove the phrase? Sorry for the :bikeshed:

Reply:
I've changed it. And don't worry about the bikeshedding, it's helpful in this case. :^)